### PR TITLE
Add Greek OSM community channels

### PR DESCRIPTION
**Pull request by comzeradd**
This is self-explanatory I think :)

Let me know if something is missing. My understanding, looking the codebase is that we checkin the `dist` build folder too, so I included it here.

### CODE REVIEW

File: resources/europe/greece/gr-mailinglist.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "gr-mailinglist",
+  "type": "mailinglist",
+  "locationSet": {"include": ["el"]},
```
Comment (bhousel) [inline resources/europe/greece/gr-mailinglist.json]: The [ISO 3166-1 code](https://en.wikipedia.org/wiki/List_of_countries_by_United_Nations_geoscheme) for Greece is actually `gr`.. The Greek language code is `el`.
(`el` does actually work here - which is nice, but we should use the correct code)

Can you change these in the files and rebuild?
```suggestion
  "locationSet": {"include": ["gr"]},
```

Comment (comzeradd) [reply resources/europe/greece/gr-mailinglist.json]: Yes, you are right. `el` is actually only the lang code. Commit amended.

Comment (comzeradd) [reply resources/europe/greece/gr-mailinglist.json]: @bhousel did you had the time to check the amendment?

Comment (bhousel) [reply resources/europe/greece/gr-mailinglist.json]: looks good, thanks for the ping 👍

**Pull request merged**